examples/text_labels_and_annotations/dfrac_demo.py

```diff
@@ -0,0 +1,15 @@
+#!/usr/bin/env python2
+# -*- coding: utf-8 -*-
+"""
+Created on Sat Feb 25 17:50:37 2017
+
+@author: watkinrt
+"""
+
+import matplotlib.pyplot as plt
+
+fig = plt.figure(figsize=(5.25, 0.75))
+fig.text(0.5, 0.3, r'\dfrac: $\dfrac{a}{b}$',
+         horizontalalignment='center', verticalalignment='center')
+fig.text(0.5, 0.7, r'\frac: $\frac{a}{b}$',
+         horizontalalignment='center', verticalalignment='center')
```

Comment:
Man, you guys are fast and thorough. I'm working on all of the requested changes.